addons/product/models/product_attribute_value.py

```diff
@@ -126,11 +134,22 @@ def unlink(self):
     def _without_no_variant_attributes(self):
         return self.filtered(lambda pav: pav.attribute_id.create_variant != 'no_variant')
 
-    def action_open_product_template_attribute_value(self):
+    def action_open_attribute_value_wizard(self):
```

Comment:
Since we have two buttons in the view, let's have one method for each button, it'll be clearer.